Documentation/git-filter-repo.txt

```diff
@@ -1039,6 +1039,15 @@ def name_callback(name):
   return name.replace(b"Wiliam", b"William")
 --------------------------------------------------
 
+The above `--name-callback` incanatation is shorthand for this raw `--commit-callback`
+
+--------------------------------------------------
+git-filter-repo --commit-callback '
+	commit.author_name = commit.author_name.replace(b"Wiliam", b"William")
+	commit.committer_name = commit.committer_name.replace(b"Wiliam", b"William")
+'
+--------------------------------------------------
```

Comment:
This is not correct; as stated in the manual: "The other callbacks operate on a small piece of the raw objects or operate on pieces across multiple types of raw object (e.g. author names and committer names and tagger names across commits and tags..."  Thus, the name-callback operations on author name and commiter name of commits AND tagger name of tags.

The message callback is similar -- it operations on commit messages of commits AND tagger messages of tags.  If someone wants to pick off just one type of name or just one type of message, they should use the commit or tag callbacks.

Reply:
Yes that makes sense - thanks for the clarification.
I think I'll let this one go then.